internal/srv/handlers.go

```diff
@@ -20,12 +20,12 @@ func (s *Server) MessageHandler(m *nats.Msg) {
 	switch msg.EventType {
 	case events.EVENTCREATE:
 		if err := s.createMessageHandler(&msg); err != nil {
-			s.Logger.Errorln("unable to process create")
+			s.Logger.Errorf("unable to process create: %s", err)
 		}
 	case events.EVENTUPDATE:
 		err := s.updateMessageHandler(&msg)
 		if err != nil {
-			s.Logger.Errorln("unable to process update")
+			s.Logger.Errorf("unable to process update: %s", err)
```

Comment:
I've been preferring zap's `w` variants. So it would be 
```suggestion
			s.Logger.Errorw("unable to process update", "error", err.Error())
```
Then the message is static (easier to parse with `jq`) and the full error is also in its own field.

Reply:
✅